scanner/matcher/matcher.go

```diff
@@ -184,3 +186,22 @@ func (m *matcherImpl) Close(ctx context.Context) error {
 	m.pool.Close()
 	return err
 }
+
+// Initialized returns nil if the matcher is fully initialized including the
+// vulnerability store.  Otherwise, an error with an explanation is returned.
+func (m *matcherImpl) Initialized() error {
+	if !m.updater.Initialized() {
+		return errors.New("the vulnerability store initial load is in progress")
```

Comment:
nit: "the vulnerability store is initializing" or "initial load for the vulnerability store is in progress"

Reply:
> the vulnerability store is initializing

This doesn't express what is holding the initialization (the initial load).

> initial load for the vulnerability store is in progress

Isn't that the same as "the vulnerability store initial load is in progress"?

Reply:
> Isn't that the same as "the vulnerability store initial load is in progress"?

yes, but the ordering of words seemed a bit awkward to me